lib/msal-common/src/request/DeviceCodeRequest.ts

```diff
@@ -8,20 +8,13 @@ import { BaseAuthRequest } from "./BaseAuthRequest";
 
 /**
  * Parameters for Oauth2 device code flow.
+ * - scopes                     - A space-separated array of scopes for the same resource.
```

Comment:
`A space-separated array` can just be `An array`, the space separation is only when the scopes are in a string.

Reply:
Actually you can match it to other requests
`Scopes the application is requesting access to.`

Reply:
Sure. Done.